### ISSUE TITLE
bad import

### ISSUE THREAD

**Issue by iplayfast** [yanniedog/crewai-autocrew#29]
```
python autocrew.py 
Traceback (most recent call last):
  File "/home/chris/ai/aiprojects/crewai-autocrew/autocrew.py", line 28, in <module>
    from core import AutoCrew
  File "/home/chris/ai/aiprojects/crewai-autocrew/core.py", line 13, in <module>
    from openai import OpenAI
ImportError: cannot import name 'OpenAI' from 'openai' (/home/chris/anaconda3/envs/crewAI/lib/python3.11/site-packages/openai/__init__.py)
```
from openai import OpenAI should be 
import openai

**Comment by iplayfast** [comment]
Ah, recent update.  For anyone else seeing this, this is the fix:
```pip install openai --upgrade```

**Comment by yanniedog** [comment]
The core problem was that the script actually wasn't installing dependencies from requirements.txt at all (until the version I've literally just released now: 2.1.5)
